contrib/php/template.py

```diff
@@ -0,0 +1,276 @@
+pkgname = "php"
+pkgver = "8.3.4"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = [
+    # common parameters
+    "--datadir=/usr/share/php",
```

Comment:
i'd also version these paths like `/usr/share/php8.3` etc

perhaps the package names too

Reply:
and we may want to have a `-default` metapackage system similar to openjdk and the likes

Reply:
Are you thinking to rename binaries as well to make it possible for multiple versions of PHP to co-exist?

Reply:
I think it's all done